src/DynamoCoreWpf/TestInfrastructure/CodeBlockNodeMutator.cs

```diff
@@ -61,7 +61,7 @@ public override bool RunTest(NodeModel node, EngineController engine, StreamWrit
 
                 DynamoViewModel.ExecuteCommand(runCancel);
             }));
-            Thread.Sleep(100);
+            Thread.Sleep(0);
```

Comment:
These sleep calls will be repurposed to allow a best effort interleave tester. Not a great way of controlling the threads, but better than nothing.

I was reducing these both to improve the performance of the test and to try and avoid masking trivial race conditions.